sklearn/utils/estimator_checks.py

```diff
@@ -360,55 +366,242 @@ def check_transformer_pickle(name, Transformer):
     assert_array_almost_equal(pickled_X_pred, X_pred)
 
 
-def check_estimators_partial_fit_n_features(name, Alg):
-    # check if number of features changes between calls to partial_fit.
-    if not hasattr(Alg, 'partial_fit'):
+def check_partial_fit_n_features(name, Estimator):
+    """Check if ValueError is raised if number of features changes between
+    multiple partial_fit calls.
+    """
+
+    if not hasattr(Estimator, 'partial_fit'):
```

Comment:
This was for the tests which use this particular function `check_partial_fit_n_features`
Actually wanted to ask you... thanks for reminding... There are a few other places where this is being tested... now with the addition of partial fit tests, do you feel they can be removed at those places...